app/views/online-class/trialClassConfirm.vue

```diff
@@ -0,0 +1,64 @@
+<template lang="pug">
```

Comment:
why is this in pug? 😅 
I don't see great benefit of using it in vue with vue's own templating features. No need to change this but I don't like mixing different templates in the same PR

Reply:
not sure, maybe this code is copy from another pug template so I update it with pug 😂